lib/ckan_client.py

```diff
@@ -0,0 +1,63 @@
+from .util import ckan_auth_api
+from .util import ckan_upload_api
+from .file import FileSystem
+
+
+class Uploader:
```

Comment:
Do we want a separate Uploader class - would we just call this something like `Client`